package-lock.json

```diff
@@ -127,57 +127,60 @@
       "peer": true
     },
     "node_modules/@angular-eslint/builder": {
-      "version": "12.5.0",
-      "resolved": "https://registry.npmjs.org/@angular-eslint/builder/-/builder-12.5.0.tgz",
-      "integrity": "sha512-abex1KdqEvUYbJqLkqhEeW4r0DuJPDzlMZYODMyBDDFTcBFYQCzLFUkhIjRNet9Vjto3Y3DbkkFUfS4DEEwjTw==",
+      "version": "12.6.1",
```

Comment:
Are the changes to `package-lock.json` intentional?

Reply:
yea npm had to make those updates to resolve peerDependency versions. I didn't try to dig into why but I did git clean the directory and try again and it was still needed.